pkgs/tools/system/consul-template/default.nix

```diff
@@ -11,6 +11,14 @@ buildGoPackage rec {
     inherit rev;
     owner = "hashicorp";
     repo = "consul-template";
-    sha256 = "15zsax44g3dwjmmm4fpb54mvsjvjf3b6g3ijskgipvhcy0d3j938";
+    sha256 = "1a1r7jwv0d4l8bcjal9chvr871hmw0ljbihgjqasp6gvjg0hfbx6";
+  };
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/hashicorp/consul-template/;
+    description = "Generic template rendering and notifications with Consul";
+    platforms = platforms.linux;
```

Comment:
Why is this linux only? It looks like it would work fine on other platforms to me.